dataclients/kubernetes/kube.go

```diff
@@ -498,9 +502,21 @@ func (c *Client) LoadAll() ([]*eskip.Route, error) {
 	c.current = mapRoutes(r)
 	log.Debugf("all routes loaded and mapped")
 
+	go c.registerSigtermHandler()
```

Comment:
of course, this is true not only for the tests :)

when someone uses skipper as an embedded library, may need to have control over the cleanup